tests/rest_api/test_configurations.py

```diff
@@ -653,26 +656,220 @@ def test_resource_hints():
+                "endpoint": {
+                    "path": "meetings/{user_id}/",
+                    "params": {
+                        "user_id": {
+                            "type": "resolve",
+                            "field": "id",
+                            "resource": "users",
+                        },
+                    },
+                },
+            },
+        ],
+    }
+    resources = rest_api_resources(config)
+
+    resource_names = [resource.name for resource in resources]
+    assert ["meetings", "user_details", "users"] == sorted(resource_names)
+
+
+def test_dependent_resource_with_multiple_parameter_bindings():
+    config: RESTAPIConfig = {
+        "client": {
+            "base_url": "https://api.example.com",
+        },
+        "resources": [
+            "users",
+            {
+                "name": "user_details",
+                "endpoint": {
+                    "path": "user/{user_id}/{group_id}",
+                    "params": {
+                        "user_id": {
+                            "type": "resolve",
+                            "field": "id",
+                            "resource": "users",
+                        },
+                        "group_id": {
+                            "type": "resolve",
+                            "field": "group",
+                            "resource": "users",
+                        },
+                    },
+                },
+            },
+        ],
+    }
+    with pytest.raises(ValueError) as e:
+        rest_api_resources(config)
+
+    error_part_1 = re.escape(
+        "Multiple resolved params for resource user_details: [ResolvedParam(param_name='user_id'"
+    )
+    error_part_2 = re.escape("ResolvedParam(param_name='group_id'")
+    assert e.match(error_part_1)
+    assert e.match(error_part_2)
+
+
+def test_one_resource_binding_two_parents():
+    config: RESTAPIConfig = {
+        "client": {
+            "base_url": "https://api.example.com",
+        },
+        "resources": [
+            "users",
+            "groups",
+            {
+                "name": "user_details",
+                "endpoint": {
+                    "path": "user/{user_id}/{group_id}",
+                    "params": {
+                        "user_id": {
+                            "type": "resolve",
+                            "field": "id",
+                            "resource": "users",
+                        },
+                        "group_id": {
+                            "type": "resolve",
+                            "field": "id",
+                            "resource": "groups",
+                        },
+                    },
+                },
+            },
+        ],
+    }
+
+    with pytest.raises(ValueError) as e:
+        rest_api_resources(config)
+
+    error_part_1 = re.escape(
+        "Multiple resolved params for resource user_details: [ResolvedParam(param_name='user_id'"
```

Comment:
We always have the same error message when we have multiple resolved parameters in a child resource – no matter if all come from the same parent or from different parents. I think that's OK.

Reply:
Yes, I think so too